### DISCUSSION TITLE
Players outside network not able to connect

### DISCUSSION THREAD

**Discussion by ImmonentDarth** [Ch4r0ne/ARK-Ascended-Server-Manager#37] [Q&A]
We are using this tool to setup a dedicated server and are able to connect from within the LAN, but players outside the LAN are unable to connect.  We have verified port-forwarding on the router and have setup firewall settings as well.  We've also disabled firewall and just had the port-forwarding.  Have you or anyone else verified players outside the LAN are able to connect?

**Comment by jugularvein** [comment]
I forwarded the ports in my router firewall and changed two Windows 10 firewall settings in Incoming which set to block connections on Private network to Allow and a friend was able to connect from his house in another state. I don't remember being prompted at during install or any program launches to allow the program, but that is what kept him from connecting. He would get auth prompt to type in a password, but then timeout after that. If you put a password on your server and your friend gets prompted and then times out, you are likely having the same issue I was.

**Comment by Ch4r0ne** [comment]
If you still have the problem, we could have a quick look together (Discord ch4r0ne)

Based on the description, I suspect a port forwarding issue in the router
